content/talks/resilient-systems.rst

```diff
@@ -0,0 +1,112 @@
+public: yes
+speakers: [miriam]
+headline:
+  - type: Training & Consulting
+    tagline: |
+      Upgrade your app with resilient systems training, tooling & consulting
+image:
+  - src: 'talks/miriam.jpg'
+event_type: workshop
+grid_template: true
+brag: |
+  Upgrade your app with resilient systems
+  training, tooling & consulting
+modules:
+  - title: Resilient & Universal Code
+    summary: |
+      Browser & device support
+      don't have to be difficult and expensive.
+      We'll cover a number of techniques
+      for building more resilient and universal
+      front end code,
+      while taking advantage of new browser features
+      as they become available.
+  - title: Intrinsic & Responsive Layouts
+    summary: |
+      The web layout system has come a long way --
+      with grids, flexbox, alignment,
+      and intrinsic sizing.
+      We'll cover the system as a whole,
+      and when to use each technique
+      to achieve dynamic & resilient layouts fast.
+  - title: Naming & Organizing Conventions
+    summary: |
+      From OOCSS to ITCSS, SMACSS, BEM, and atomic design,
+      there are a number of naming and organizing
+      conventions that can help us write
+      modular & maintainable CSS.
+      We'll look at what these systems have in common,
+      and how to mix-and-match for the needs
+      of a particular team or project.
+  - title: Design Systems & Component Libraries
+    summary: |
+      Reusable design & component systems
+      can improve consistency,
+      code-quality,
+      and team communication.
+      You don't need a dedicated team and budget
+      to get started.
+      We'll talk about planning and designing systems
+      in an agile and integrated environment.
+  - title: Style Guides & Documentation
+    summary: |
+      Documentation is central to maintainable code,
+      but how do we make sure the documentation
+      itself is maintained and up-to-date?
+      We'll look at workflow and tooling
+      for automated and integrated style guides
+      and documentation.
+  - title: Tooling & Testing
+    summary: |
+      The tools we use help define our workflow,
+      and the long-term reliability of our code.
+      Tools & tests should help encourage best-practice
+      without locking us into antiquated systems.
+      We'll talk about designing
+      the right tooling and testing workflow
+      for your team or project.
+  - title: Team Process & Wokflow
+    summary: |
+      Struggling with messy handoff
+      or communication between designers & developers?
+      We can help you find the
+      practical workflows that keep your
+      process running smoothly
+      to create beautiful & integrated user experiences.
+  - title: HTML & CSS in JS Frameworks
+    summary: |
+      With the advent of JS frameworks,
```

Comment:
```suggestion
      With the advent of JavaScript frameworks,
```

It strikes me that JS frameworks are not new, so maybe this could be stated in a way that more clearly says what you mean. Perhaps the "growth of modern JavaScript frameworks"?